cmux-tui/scripts/smoke-windows-conpty-resize.py

```
@@ -69,29 +82,157 @@ def wait_for_socket(
+            while not self.stop_event.is_set():
+                chunk = self._pty.read(OUTPUT_READ_CHUNK_SIZE)
+                if not chunk:
+                    self._enqueue(None)
+                    return
+                self._enqueue(chunk)
         except (EOFError, OSError) as error:
-            if not ready:
-                events.put(error)
+            if not self.stop_event.is_set():
+                self._enqueue(error)
         except BaseException as error:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `BaseException`

(BLE001)